assignments/assignment01.rb

```diff
@@ -37,14 +37,14 @@ def number_to_string(n, lang)
 
 # it accepts a string
 # and returns the same string with each word capitalized.
-def titleize(s)
-  # your implementation here
-end
 
-# Your method should generate the following results:
-titleize "hEllo WORLD"          #=> "Hello World"
+def titleize(input)
 
-titleize "gooDbye CRUel wORLD"  #=> "Goodbye Cruel World"
+  word = input.split.each { |x| x.downcase.capitalize! }.join(' ')
```

Comment:
if you use each, then you need to do something with the result of each iteration yourself. Since you want to join the result, I think you mean to use map instead of each.